### ISSUE TITLE
oc new-app uses inscrutable name in error message

### ISSUE THREAD

**Issue by deads2k** [openshift/origin#3230]
The error itself may be valid.  The Dockerfile had a line in it: `EXPOSE 111/udp `  (note the trailing space), but the error message is listing the value of the `FROM`, not the value of the imagestream that is created which seems more reasonable: `imagestreams/docker-nfs-server`.

```
[deads@deads-dev-01 openshift-docs]$ oc new-app https://github.com/deads2k/docker-nfs-server --strategy=docker -o  yaml
error: can't setup "ubuntu-debootstrap:14.04": failed to parse port "": strconv.ParseInt: parsing "": invalid syntax
```

**Comment by bparees** [comment]
@oatmealraisin this is super old so first confirm you can recreate the problem, if not we can just close it.

**Comment by oatmealraisin** [comment]
@bparees Not able to recreate, current output looks correct
http://pastebin.com/00zcBW63

**Comment by bparees** [comment]
@oatmealraisin i'm guessing @deads2k has corrected the typo in the dockerfile in his repo, you'll have to create a fork of his repo and "break" the dockerfile to attempt your recreate.

**Comment by oatmealraisin** [comment]
Good point.  I reset to the 2 commits previous to this issue, and checked both for the trailing space after ```EXPOSE 111/udp``` .
I also tried adding other whitespace to the file, which doesn't seem to do anything either.  Maybe someone added a trim in the parser?  Should I roll back to the 6/16/15 version of openshift?

**Comment by bparees** [comment]
@oatmealraisin nah, but maybe try actually corrupting the dockerfile in another way so it really doesn't parse.  eg add a non-docker command like "RUN2" to the dockerfile.

**Comment by oatmealraisin** [comment]
I've managed to get some output similar to what he was talking about.  If you give new-app a bogus file it complains about the FROM section.  I gave it a .jpg and it outputs . . .

[vagrant@localhost ~]$ oc new-app https://github.com/oatmealraisin/docker-nfs-server --strategy=docker -o  yaml > not_broken.txtA
error: the Dockerfile in the repository "/tmp/gen260586057" has no FROM instruction

However, it seems like this is pretty fitting.  I've noticed that most sections are omitted if they are corrupted, unless they're critical like FROM.

@bparees close issue?
